addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -5,7 +5,17 @@
             <t t-if="messageInReplyToView">
                 <t t-if="!messageInReplyToView.messageView.message.parentMessage.isEmpty">
                     <b class="o_MessageInReplyToView_author text-muted ml-2">@<t t-esc="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
-                    <span class="o_MessageInReplyToView_body ml-1" t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody" t-on-click="messageInReplyToView.onClickReply"/>
+                    <span class="o_MessageInReplyToView_body ml-1" t-on-click="messageInReplyToView.onClickReply">
+                        <t t-if="messageInReplyToView.hasBodyBackLink">
+                            <t t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody"/>
+                        </t>
+                        <t t-if="messageInReplyToView.hasAttachmentBackLink">
+                            <span class="font-italic">
+                                Click to see the attachments
+                            </span>
```

Comment:
To follow on the comment from Alex, if we want to avoid spacing from text node, then span should not contain new lines or other extra spaces on it, otherwise the purpose is defeated. Make sure to add margin as necessary either on the text or on the icon.